src/libtiled/mapreader.cpp

```diff
@@ -391,8 +391,15 @@ SharedTileset MapReaderPrivate::readTileset()
                                       tileSpacing, margin);
 
             tileset->setColumnCount(columns);
-            tileset->setCanRotate(atts.value(QLatin1String("canrotate")).toInt());
-            tileset->setAlternateRotation(atts.value(QLatin1String("alternaterotation")).toInt());
+            // recognize older file format for WangTile rotation
```

Comment:
This is rather pointless since only you and me are using the older file format, right? :-)

Reply:
Oh, I know of at least a third person, and more people might have downloaded the tileset from my github repository ... I agree that it should be a temporary measure.